src/modules/esl-a11y-group/core/esl-a11y-group.ts

```diff
@@ -0,0 +1,97 @@
+import {ExportNs} from '../../esl-utils/environment/export-ns';
+import {ESLBaseElement, attr, boolAttr} from '../../esl-base-element/core';
+import {bind} from '../../esl-utils/decorators/bind';
+import {ARROW_DOWN, ARROW_LEFT, ARROW_RIGHT, ARROW_UP} from '../../esl-utils/dom/keys';
+import {TraversingQuery} from '../../esl-traversing-query/core/esl-traversing-query';
+
+export type GroupTarget = 'next' | 'prev' | 'current';
+
+@ExportNs('A11yGroup')
+export class ESLA11yGroup extends ESLBaseElement {
+  public static is = 'esl-a11y-group';
+
+  @attr({defaultValue: '::child'}) public targets: string;
```

Comment:
Minor: add attribute change handler for targets

Reply:
There is no cache of $targets, so they are collected every time.